webservices/resources/reports.py

```diff
@@ -1,10 +1,12 @@
 import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
```

Comment:
It's to print out queries that error when you attempt to do a simple `print(query)` for debugging.  That reminds me, I've written this about three times so I'll take them method I used for that print statement and pull it up into the utils file.